edu.wpi.first.wpilib.plugins.cpp/src/main/resources/cpp-zip/ant/build.xml

```diff
@@ -147,4 +151,58 @@
 	</assert>
 	<echo>roboRIO image version validated</echo>
   </target>
+  
+  <!-- libs.name should not contain spaces as it is used to name a file -->
+  <macrodef name="deploy-libs">
+    <attribute name="libs.name"/>
+    <attribute name="libs.basedir"/>
+    <element name="libs.local"/>
+    <sequential>
+      <local name="libs.local.notEmpty"/>
+      <local name="libs.local.checksum"/>
+      <local name="libs.deployed.checksum"/>
+      <local name="libs.local.modified.property"/>
+      
+      <delete file="@{libs.basedir}/@{libs.name}.properties"/>
+      <scp file="${adminUsername}@${target}:${libDeploy.dir}/@{libs.name}.properties" 
+           todir="@{libs.basedir}"
+           password="${adminPassword}" 
+           trust="true"
+           failonerror="false"/>
+      <restrict id="libs.local.modified">
+        <libs.local/>
+        <modified update="true"
+                      seldirs="true"
+                      cache="propertyfile"
+                      algorithm="digest"
+                      comparator="equal">
+          <param name="cache.cachefile"     value="@{libs.basedir}/@{libs.name}.properties"/>
```

Comment:
Some odd formatting--spaces before `value=` on this line, and the `modified` additional attributes aren't aligned with the `update="true"` like other places in this file (`scp`, `sshexec`)